repo2docker/buildpacks/nix/nix-shell-wrapper

```diff
@@ -8,8 +8,13 @@ _term() {
 
 trap _term SIGTERM
 
-echo "$*"
-nix-shell default.nix --command "$*" &
+echo 'command' "$*"
```

Comment:
Do you have thoughts on `$@` vs `$*`? In `start` we use `$@`

Reply:
Nope in fact I believe that `$@` is the recommended way